test/ci/env/oxygen.env

```diff
@@ -3,4 +3,8 @@
 #
 ANT_VERSION=1.10.9
 SAXON_VERSION=10.6
```

Comment:
If this file is supposed to represent the "Latest oXygen" then I should make it match 25.1. Do you agree?

Reply:
Updated with 656cc20748cf4b1a2b78adfc62557896e417d9e3. Oxygen 25.1 uses Saxon 11.4 and XML Calabash 1.5.3-110, according to the release notes for versions [25.1](https://www.oxygenxml.com/xml_editor/whats_new.html#25.1Component_Updates) and [25.0](https://www.oxygenxml.com/xml_editor/whatisnew25.0.html#25.0Component_Updates).